client/driver/rkt.go

```diff
@@ -240,7 +243,20 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	img := driverConfig.ImageName
 
 	// Build the command.
-	cmdArgs := make([]string, 0, 32)
+	cmdArgs := make([]string, 0, 50)
+
+	// Run rkt with env command to inject PATH as rkt needs to be able to find iptables
+	envAbsPath, err := GetAbsolutePath(envCmd)
```

Comment:
Unclear why we can just use the option `-set-env` and just run Rkt with all environment variables minus those in the blacklist passed through?